packages/middleware-token/src/tokenDefaultProvider.ts

```diff
@@ -0,0 +1,3 @@
+import { nodeProvider } from "@aws-sdk/token-providers";
+
+export const tokenDefaultProvider = (input: unknown) => nodeProvider(input as any);
```

Comment:
Can this be a direct reference and preserve the function signature?

Reply:
Added direct reference in [`2af788e` (#4234)](https://github.com/aws/aws-sdk-js-v3/pull/4234/commits/2af788e106ab8761a0edb572a43fddd998334cb8)